package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "iin-checker",
   "description": "Issuer identification number checker which returns details about a credit/debit card",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
I left it at 0.3.0 as it was not breaking change, but has got a lot of changes can update to 1.0.0